File: cb_test/py_test/digital.py
import wiringpi2
import sys

OUTPUT = 1
CHECK = 6
pin =1

wiringpi2.pinMode(pin,OUTPUT)
Mo = wiringpi2.pinMode(pin,CHECK)
MS = 1000

print "**********************"
print "delay:%s ms,Mode:%s" %(MS,Mo)
print "**********************"

while 1:
	wiringpi2.digitalWrite(pin,1)
	wiringpi2.delay(MS)
	wiringpi2.digitalWrite(pin,0)
	wiringpi2.delay(MS)

File: cb_test/c_test/digital.c
/**
 * softPwm test 
 * author:gootoomoon 
 * gcc -o test digital.c -lwiringPi -lpthread
 */
#include <stdio.h>
#include <errno.h>
#include <string.h>

#include <wiringPi.h>
#define OUTPUT 	1
#define HIGH 	1
#define LOW 	0

int pin = 1;

int main()
{
	if(wiringPiSetupPhys() == -1)
	{
		fprintf(stdout, "wiring lib init error:%s", strerror(errno));	
		return 1;
	}
	pinMode(pin,OUTPUT);

	for(;;){
		digitalWrite(pin,HIGH);
		delay(1000);
		digitalWrite(pin,LOW);
		delay(1000);
	}
}
